electron/launcher.ts

```diff
@@ -502,10 +514,23 @@ async function runLegendaryOrGogdlCommand(
       return { stdout, stderr, fullCommand: safeCommand }
     })
     .catch((error) => {
-      errorHandler({ error, logPath: options?.logFile })
+      if (error === 'MemoryError:') {
+        logWarning(
+          'Install failed, trying again with higher limit',
+          LogPrefix.Legendary
+        )
+        return { stdout: '', stderr: '', fullCommand: safeCommand, error }
+      }
```

Comment:
This should be in `LegendaryGame.install` instead

Reply:
It is more complicated than that. 
Right now the install command fails silently, the promise is not rejected since the message from legendary is not an error.
So I needed a workaround and the only time I get this info, the MemoryError is on the `launchCommand` 
Dont know if that's what you mean

Reply:
After running the command, you get `stdout` and `stderr`. You can then check for the "MemoryError" string (or anything else you'd like) there, instead of doing it for every legendary/gogdl command

Reply:
IMO, when this function should return an error, it should mean "There was an error running the command", not "There was an error within the program that we were running"
Imagine adding more runners which have similar messages like this, over time you'd just have a giant list of "does the output contain this, then reject with this". That's not scalable

Reply:
I understand, but like I said this is not an error and the command, even though, fails, it returns a success exit code.
I believe that this memory management could be good to be done on Legendary but maybe there is a reason for that.

I can try to think on another way but it was not that simple to come to a solution for that since we need to intervene on the command to call it again with different parameters

Reply:
Ok, I fixed that now, the error is handled on Legendary.install like you said 👍🏽